pom.xml

```diff
@@ -628,6 +628,11 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Why add this depend？

Reply:
mockito-inline can enable `mockStatic` for unit test, this can make unit test more simple and clean